src/app/components/core/logs/system-log/system-log.component.html

```diff
@@ -79,11 +79,11 @@
           <div class="column">
             <label class="label is-small">Search</label>
             <input type="search" class="input is-fullwidth is-small" placeholder="search text on current page"
-              [(ngModel)]="searchTerm" />
+              [(ngModel)]="searchTerm" id="syslog-search" (keyup)="getFilteredSysLog()"/>
```

Comment:
Don't we need the css classes?

Keep the placeholder, just remove "on current page"